src/Command/RunCommand.php

```diff
@@ -193,12 +181,8 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 $job->getComponentId(),
                 $job->getProjectId()
             ));
-            $options = (new ClientOptions())
```

Comment:
Přesunul jsem do factory, už toho bylo dost co se tomu nastavuje

Reply:
nevím proč do factory, je factory to options pro factory pro klienta, to už začíná byt dost zamotaný a navíc se to používá na jednom místě
afaik by stačila metoda (klidně statická, kdybys ji chtěl otestovat a nesrat se s vytvářením instance)

Reply:
Tak z toho udelam helper se statickou metodou no 🤷

Reply:
Nicmene jsem vychazel z toho ze `CreditsCheckerFactory` je taky factory, dokonce tvoje prace.

Reply:
no to je zaprvé vykopírovaný z docker bundle, za druhé k něčemu jinýmu a za třetí je to kardinálně nahovno 😆